providers/hns/hns_roce_u_trace.h

```diff
@@ -121,13 +121,19 @@ LTTNG_UST_TRACEPOINT_EVENT(
 
 #include <lttng/tracepoint-event.h>
 
-#else
+#elif defined(USDT_ENABLED)
 
 #ifndef __HNS_TRACE_H__
 #define __HNS_TRACE_H__
 
+#include <util/usdt.h>
+
+#define rdma_tracepoint(arg...) USDT(arg)
```

Comment:
nit: Could be nice if this is defined only once.

Reply:
FWIW, I followed the existing pattern from LTTNG tracing, where each provider provides self-contained definitions.